modules/backup/main.tf

```diff
@@ -178,7 +178,7 @@ resource "google_storage_bucket_iam_member" "sql_instance_account" {
   count  = var.enable_export_backup ? 1 : 0
   bucket = split("/", var.export_uri)[2] #Get the name of the bucket out of the URI
   member = "serviceAccount:${data.google_sql_database_instance.backup_instance.service_account_email_address}"
-  role   = "roles/storage.objectCreator"
+  role   = "roles/storage.objectUser"
```

Comment:
Instead of replacing `roles/storage.objectCreator` with `roles/storage.objectUser`, I would suggest adding another IAM binding for `roles/storage.objectViewer` to give the SA read permissions.

Reply:
I think this is better option but I don't understand why it's needed in first place. Workflow is only exporting to the bucket. It does not list or try to get any info about anything else on bucket.

Reply:
I don't know why it fails with `permissions missing are: [storage.objects.delete]` , hence objectUser permissions

agree with creating another IAM binding